var/spack/repos/builtin/packages/espresso/package.py

```diff
@@ -91,38 +94,55 @@ def install(self, spec, prefix):
             options.append('--enable-openmp')
 
         if '+scalapack' in spec:
-            options.append('--with-scalapack=yes')
+            scalapack_option = 'intel' if '^intel-mkl' in spec else 'yes'
+            options.append('--with-scalapack={0}'.format(scalapack_option))
 
         if '+elpa' in spec:
-            if spec.satisfies('@:5.4.0'):
```

Comment:
p.s. hopefully all developers are convinced one day NOT to download dependencies themselves and use sane build systems/flags....